### ISSUE TITLE
[Bug]: GenericFailure: should send result success

### ISSUE THREAD

**Issue by chneau** [web-infra-dev/rspack#8513]
### System Info

Docker: `oven/bun:alpine`
Bun: 1.1.36
"@rsbuild/core": "^1.1.4",
"@rsbuild/plugin-react": "^1.0.7",

### Details



                                                                                                                                                                                                                                                                           
 > [6/6] RUN RUST_BACKTRACE=full bun run build:                                                                                                                                                                                                                                  
0.209 $ rsbuild build                                                                                                                                                                                                                                                            
0.419   Rsbuild v1.1.4                                                                                                                                                                                                                                                           
0.419                                                                                                                                                                                                                                                                            
0.830 error                                                                                                                                                                                                                                                                      
0.830  code: "GenericFailure"
0.830 
1.152 Panic occurred at runtime. Please file an issue on GitHub with the backtrace below: https://github.com/web-infra-dev/rspack/issues
1.152 Message:  failed to send error message: SendError { .. }
1.152 Location: ./crates/rspack_core/src/utils/task_loop.rs:90
1.152 
1.152 Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
1.152 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
1.152 <empty backtrace>
2.333 error: script "build" was terminated by signal SIGABRT (Abort)


### Reproduce link

_No response_

### Reproduce Steps

1. Run build in Dockerized environment
2. It's no more working since a certain version, I'll try to pin point it

**Comment by chneau** [comment]
It's working on `"@rsbuild/core": "=1.1.3",` but not anymore in `"@rsbuild/core": "=1.1.4",`

**Comment by chneau** [comment]
For some reason I can't get the error to be more explicit again but I have it here:

```
1.018  code: "GenericFailure"
1.018 
1.019 Panic occurred at runtime. Please file an issue on GitHub with the backtrace below: https://github.com/web-infra-dev/rspack/issues
1.019 Message:  should send result success: (ExecuteModuleResult { error: None, cacheable: true, file_dependencies: {}, context_dependencies: {}, missing_dependencies: {}, build_dependencies: {}, code_generated_modules: {}, assets: {}, id: 0 }, {}, {Identifier(u!("commonjs|/app/node_modules/@rsbuild/core/compiled/css-loader/index.js??ruleSet[1].rules[1].use[1]!builtin:lightningcss-loader??ruleSet[1].rules[1].use[2]!/app/node_modules/antd/dist/reset.css"))}, [])
1.019 Location: crates/rspack_core/src/compiler/module_executor/execute.rs:315
1.019 
1.019 Backtrace omitted.
```

**Comment by chneau** [comment]
Pinning to `"@rsbuild/core": "=1.1.3"` seems to be a good work around for now.

**Comment by JSerFeng** [comment]
There are some issues of bun, https://github.com/oven-sh/bun/issues/15383 I'll check this out after they fixed it

**Comment by chenjiahan** [comment]
Should be resolved in Bun 1.1.39, see https://github.com/oven-sh/bun/issues/15383
